experimental/Schemes/ToricDivisorOnScheme.jl

```diff
@@ -0,0 +1,181 @@
+########################################################################
+# Toric divisors on (toric) schemes
+#
+# This is a wrapper for the `ToricDivisor` functionality to port them 
+# to the realm of `ToricCoveredScheme`s. All the functionality is lazy 
+# and should by default only trigger computations on the toric side. 
+# However, despite inheriting all those methods, the actual type fits 
+# within the scheme's framework and the truely geometric objects are 
+# derived and cached via the respective calls. In particular, these 
+# divisors can seamlessly be used with usual divisors. 
```

Comment:
@HereAround , @lkastner : 
This is a draft for how to hopefully facilitate seamless wrappers for `ToricDivisor`s. You can try it out by modifying another file (I will write it below). 

Let me know what you think.